ee/tables/dism_default_associations/dism_default_associations.go

```diff
@@ -1,7 +1,7 @@
 //go:build windows
 // +build windows
 
-package dsim_default_associations
+package dism_default_associations
```

Comment:
I corrected this spelling, then later remembered Becca having the same issue with it. Should I just put it back?